prov/verbs/src/ep_rdm/verbs_tagged_ep_rdm_states.c

```diff
@@ -283,10 +283,10 @@ fi_ibv_rdm_tagged_rndv_rts_send_ready(struct fi_ibv_rdm_tagged_request *request,
 	header->base.tag = request->minfo.tag;
 	header->base.service_tag = 0;
 	header->total_len = request->len;
-	header->src_addr = (uint64_t) (uintptr_t) request->src_addr;
+	header->src_addr = (uintptr_t)request->src_addr;
 
-	header->id = request;
-	request->rndv.id = request;
+	header->id = (uintptr_t)request;
+	request->rndv.id = (uintptr_t)request;
```

Comment:
@a-ilango now, all pointers in scope of this function castes to uintptr_t then assignments to the destination variable which has also unsigned type with size not less than uintptr_t. So, I would prefer to change everything to this form in the future, probably in separated PR with refactoring. What's your opinion?